src/collective/behavior/talcondition/utils.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-from App.class_init import InitializeClass
```

Comment:
If this package really supports Plone 4.3 I think it should be:
```
try:
    from AccessControl.class_init import InitializeClass
except ImportError:
    from App.class_init import InitializeClass
```
Also please add a note to:
https://github.com/collective/collective.behavior.talcondition/blob/master/CHANGES.rst

Reply:
Ok, thanks. 

It's done